packages/jest-reporters/src/index.ts

```diff
@@ -14,12 +14,6 @@ import {
   trimAndFormatPath,
 } from './utils';
 
-export type {Config} from '@jest/types';
-export type {
-  AggregatedResult,
-  SnapshotSummary,
-  TestResult,
-} from '@jest/test-result';
```

Comment:
Breaking change. I think there is no need of these reexports. Only types, hence can be installed as devDependecies and that’s fine.

Reply:
The thought was you only need the reporter dependency to get all the types you need to write a reporter, instead of having to dig out where the types come from